signalfx-org-metrics/metrics.yaml

```diff
@@ -1,3 +1,255 @@
+sf.org.apm.numSpansDroppedThrottle:
+  brief: The number of spans SignalFx receives beyond the allowed ingest volume.
+  description: 'The number of spans SignalFx receives beyond the allowed ingest volume.
+    SignalFx drops spans it receives after the ingestion volume limit is reached.
+
+    Dimension(s): `orgId`
+    
+    Data resolution: 1 second'
+  metric_type: counter
+  title: sf.org.apm.numSpansDroppedThrottle
+
+sf.org.apm.grossSpansReceived:
+  brief: The number of spans SignalFx receives before filtering and throttling.
+  description: 'The number of spans SignalFx receives before filtering or throttling.
+
+  Dimension(s): `orgId`
+
+  Data resolution: 1 second'
+  metric_type: counter
+  title: sf.org.apm.grossSpansReceived
+
+sf.org.apm.grossSpansReceivedByToken:
+  brief: The number of spans SignalFx receives for a specific access token
+    before filtering and throttling.
+  description: 'The number of spans SignalFx receives for a specific access token
+    before filtering or throttling.
+
+    Dimension(s): `orgId`, `tokenId`
+    
+    Data resolution: 1 second'
+  metric_type: counter
+  title: sf.org.apm.grossSpansReceivedByToken
+
+sf.org.apm.numSpansReceived:
+  brief: The number of spans SignalFx receives after filtering and throttling.
+  description: 'The number of spans SignalFx receives after filtering and throttling.
+
+  Dimension(s): `orgId`
+    
+  Data resolution: 1 second'
+  metric_type: counter
+  title: sf.org.apm.numSpansReceived
+
+sf.org.apm.numSpansDroppedThrottleByToken:
+  brief: The number of spans SignalFx receives for a specific access token
+    beyond the allowed ingest volume.
+  description: 'The number of spans SignalFx receives for a specific access token
+    beyond the allowed ingest volume. SignalFx drops spans it receives after
+    the ingestion volume limit is reached.
+
+    Dimension(s): `orgId`, `tokenId`
+    
+    Data resolution: 1 second'
+  metric_type: counter
+  title: sf.org.apm.numSpansDroppedThrottleByToken
+
+sf.org.apm.numSpansReceivedByToken:
+  brief: The number of spans SignalFx receives for a specific access token
+    after filtering and throttling.
+  description: 'The number of spans SignalFx receives for a specific access token
+    after filtering and throttling.
+
+  Dimension(s): `orgId`, `tokenId`
+    
+  Data resolution: 1 second'
+  metric_type: counter
+  title: sf.org.apm.numSpansReceivedByToken
+
+sf.org.apm.grossSpanBytesReceived:
+  brief: The number of bytes SignalFx receives for a span after decompression
```

Comment:
Not really "for a span". Maybe "The number of bytes SignalFx receives from spans, after decompression but before filtering and throttling"